CMakeLists.txt

```diff
@@ -248,12 +248,14 @@ add_subdirectory(sqliteimpl)
 add_subdirectory(tests)
 add_subdirectory(templates/security)
 add_subdirectory(docs/api)
+add_subdirectory(examples/cmake)
 if (${BUILD_CLI})
   add_subdirectory(clicache)
   add_subdirectory(plugins/SQLiteCLI)
 endif()
 
-install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/examples/ DESTINATION examples)
+install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/examples/ DESTINATION examples
```

Comment:
You don’t want to install then .in files. This should be the configured files in the binary dir now.